### PR TITLE
add option FAST_FLOAT to cmake build

### PR DESCRIPTION
**Pull request by zdenop**
Add new cmake option -DFAST_FLOAT=ON for faster LSTM with float

### CODE REVIEW

**Comment by zdenop** [comment]
I still wonder why there are other commits than 3c6bc12 in this PR ...
I just did:
```
git pull upstream master
git checkout -b cmake.float
patch -p 1 <cmake.float.patch
git add CMakeLists.txt cmake/Configure.cmake 
git commit -m "Add new cmake option -DFAST_FLOAT=ON for faster LSTM with float" 
git push
```
and then send PR via github...

File: CMakeLists.txt [hunk 1]
```diff
@@ -85,6 +85,7 @@ option(OPENMP_BUILD "Build with openmp support" OFF)  # see issue #1662
 option(GRAPHICS_DISABLED "Disable disable graphics (ScrollView)" OFF)
 option(DISABLED_LEGACY_ENGINE "Disable the legacy OCR engine" OFF)
 option(ENABLE_LTO "Enable link-time optimization" OFF)
+option(FAST_FLOAT "Enable link-time optimization" OFF)
```
Comment (stweil) [inline CMakeLists.txt:88]:
```suggestion
option(FAST_FLOAT "Enable float for LSTM" OFF)
```

**Comment by stweil** [comment]
Thanks. I squashed all commits and fixed the description text for `FAST_FLOAT`.

File: CMakeLists.txt (hunk 1 above)
Comment (zdenop) [reply CMakeLists.txt]: Thanks ;-)

**Pull request merged**